O21.Game/Game.fs

```diff
@@ -60,7 +60,7 @@ type Game(window: WindowParameters, content: LocalContent, data: U95Data) =
 module GameLoop =
     let Run (lifetime: Lifetime, window: WindowParameters) (content: LocalContent, data: U95Data): unit =
         let game = Game(window, content, data)
-        let musicPlayer = CreateMusicPlayer lifetime (content.SoundFontPath, data.MidiFilePath)
+        let musicPlayer = CreateMusicPlayerAsync lifetime (content.SoundFontPath, data.MidiFilePath) |> Async.RunSynchronously
```

Comment:
Nope, `Async.RunSynchronously` is no-no, and there's no point in using it here.

The point was to not block the game loop during the loading, and this still causes it to be blocked.

Ideally, you should spawn the loading task here and pass it to the `musicPlayer` object, so it somehow tracks the task, and starts playing the music whenever the task is ready.

Reply:
Fixed it, now the **MusicPlayer** is initialized in another task